lib/mw/log.js

```diff
@@ -1,4 +1,33 @@
 var UUID = require('libuuid');
+var winston = require('winston');
+require('winston-redis').Redis;
+
+winston.emitErrs = true;
+
+var logger = new winston.Logger({
+  transports: [
+    new winston.transports.File({
+      level: 'info',
+      handleExceptions: true,
+      json: true,
+      maxsize: Config.get('logger:maxSize'),
+      maxFiles: Config.get('logger:maxFiles'),
+      filename: Config.get('logger:filename'),
+      colorize:false
+    }),
+    new winston.transports.Console({
+      level: 'debug',
+      handleExceptions: true,
+      json: false,
+      colorize: true
+    }),
+    new winston.transports.Redis({
```

Comment:
Any reason for piping to Redis by default? What kind of rotation/limits does this transport implement?